flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptivebatch/ExecutionPlanSchedulingContext.java

```diff
@@ -58,4 +58,11 @@ int getConsumersMaxParallelism(
      * @return the number of pending operators.
      */
     int getPendingOperatorCount();
+
+    /**
+     * Retrieves the JSON representation of the stream graph for the original job.
+     *
+     * @return the JSON representation of the stream graph.
+     */
+    String getJsonStreamGraph();
```

Comment:
getStreamGraphJson()

Add more comments to clarify the JSON pattern if a job graph is submitted.

Reply:
Updated.